src/main/java/org/folio/circulation/resources/renewal/RenewalStrategy.java

```diff
@@ -0,0 +1,12 @@
+package org.folio.circulation.resources.renewal;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.circulation.resources.context.RenewalContext;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.Result;
+
+public interface RenewalStrategy {
```

Comment:
What are the options for renewal, either regular or overridden?

Reply:
Yes, override or usual renewal.

Reply:
Thanks